mod_app/migrations/0023_alter_bibliographyitem_full_citation_and_more.py

```diff
@@ -0,0 +1,90 @@
+# Generated by Django 4.2.5 on 2024-06-27 13:55
+
+import ckeditor.fields
+import ckeditor_uploader.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("mod_app", "0022_bibliographyitem_annotation"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="bibliographyitem",
+            name="full_citation",
+            field=ckeditor.fields.RichTextField(
+                help_text="Please use Harvard reference list style. Examples: Author surname, initial. (year). 'Title of article/book/website/photograph'. Available at: URL or DOI where applicable (date accessed). Please note that the website or photo title should be italicised. For more detailed information and examples visit: https://www5.open.ac.uk/library/referencing-and-plagiarism/quick-guide-to-harvard-referencing-cite-them-right",
+                null=True,
+            ),
+        ),
+        migrations.AlterField(
+            model_name="bibliographyitem",
+            name="short_citation",
+            field=models.CharField(
+                help_text="Please use Harvard in-text citation style. Examples: (Author surname, year), or in the case of up to 3 authors: (Author 1 surname, Author 2 surname, Author 3 surname, year), or in the case or more than 3 authors: (Author 1 surname, et al., year) ",
+                max_length=200,
+                null=True,
+            ),
+        ),
```

Comment:
these look the same as 
https://github.com/UCL-ARC/museum_of_dreams/blob/90db50e9bd558b0953f76a421667512cb35c3120/mod_app/migrations/0017_alter_help_text_for_citations.py#L14-L30
However, the previous migrations include two spaces after full stops whereas the new migrations have a single space

Reply:
We could add a check in CI to make sure there are no missing migrations:
```python
python manage.py makemigrations --check --dry-run --no-input
```
which I think would fail if there are any migrations missing. If this runs on every PR then it should help avoid merging anything with missing migrations